### ISSUE TITLE
Adding Manifest.xml for dual mod compatibilities?

### ISSUE THREAD

**Issue by NinjaSiren** [AthenaSulisMinerva/CombatExtendedFastTrack#182]
I have made a simple Manifest.xml for this mod to be compatible with Fluffy's Mod Manager.
[Manifest.zip](https://github.com/AthenaSulisMinerva/CombatExtendedFastTrack/files/3908351/Manifest.zip)

**Comment by AthenaSulisMinerva** [comment]
This most likely won't work, because the `-FT-011` part in your Manifest.xml violates the following requirement in the [documentation for Mod Manager](https://github.com/fluffy-mods/ModManager/blob/master/ForModders.md#version):

> Versions may only contain numbers, separated by periods, in the format `major.minor.build.revision`. Alphabetical characters (e.g. `v1.0` or `0.2-beta`) are **not allowed**.

I discussed this with @sumghai on Hangouts, and we concluded that we can't change our version numbering scheme to drop the `-FT-` part, because we use that to tell people what base CE version our latest FastTrack update is built upon.

Even if we did something like `1.0.1.8.2.011`, that goes against the standard `major.minor.build.revision` format anyway simply by having more than 4 elements. Besides, the `FT-011` suffix runs independently of whatever the main version numbers are.

Finally, if we come up with our own, unique version numbers (e.g. `1.8.011`), that would just confuse the hell out of our users.

In short, we originally removed ModSync and FMM support for these reasons, so we're not adding them back in.
